src/app/src/components/FilterSidebar.jsx

```diff
@@ -74,16 +75,18 @@ const filterSidebarStyles = theme =>
         },
         filterButton: {
             backgroundColor: theme.palette.action.main,
-            color: '#000',
+            '&:hover': {
+                backgroundColor: theme.palette.action.dark,
+            },
+            color: theme.palette.secondary.contrastText,
```

Comment:
We don't have to calculate the secondary text color; MUI assigns it to this field automatically, since 'secondary' is a known text field